.gitignore

```diff
@@ -50,16 +51,17 @@ coverage.xml
 /CACHE
 
 # Sphinx documentation
-docs/_build/
+docs / _build/
 
 # PyBuilder
 target/
 
 # IDE
 .idea/
+.DS_Store
 
-# External Libraries 
-wger/core/static/bower_components
+# External Libraries
+wger / core / static / bower_components
```

Comment:
@Manorlds-Eaglespark, why did you have to change the path for these `dir` in this file here.
I think you should set them back for consistency.

Reply:
These were reverted. Thanks